### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by mcourtot** [EBISPOT/DUO#90]
volunteers for french translation?

Francis
Melanie

**Comment by mullejea** [comment]
Hello my dear Mélanie :)
I would be delighted to help here.

**Comment by FrancisJMR** [comment]
Hi all, I've done a first pass translation. Please find it here:

Salut à toutes et tous, j'ai fait une première traduction. Veuillez la trouver ici:

https://docs.google.com/spreadsheets/d/1Oc30NRXsqVROqMn9KG943WyE-hx3drdR9xrg5Uvsjf0/edit?usp=sharing

Please feel free to edit, or create a new sheet if we'd like to iterate on versions!

Best,

Francis

**Comment by mullejea** [comment]
Ok great. looks already good. I might add a line below a line for which I think something might be improved, if ok for you?

**Comment by FrancisJMR** [comment]
Yes absolutely, I did a first translation but input and revisions are important. Thank you.

**Comment by mullejea** [comment]
Thanks. My modifications have been added on your document. Hope they are useful. Let me know if anything ,eeds to be discussed.

**Comment by FrancisJMR** [comment]
Jean: it looks like the version is ready for a first review by Melanie and others?

Melanie, do you want to take a look and let us know if we should engage with more people to validate the french version?
(Same link as above (https://docs.google.com/spreadsheets/d/1Oc30NRXsqVROqMn9KG943WyE-hx3drdR9xrg5Uvsjf0/edit?usp=sharing)[https://docs.google.com/spreadsheets/d/1Oc30NRXsqVROqMn9KG943WyE-hx3drdR9xrg5Uvsjf0/edit?usp=sharing])
